doc/tech/glossary.md

```diff
@@ -50,8 +50,8 @@ To help guide the reader, this glossary provides an introduction to the terminol
 - **NFVI Software Profile (NFVI SW Profile)**: defines the behaviour, capabilities and metrics provided by an NFVI Software Layer
 - **NFVI Software Configuration (NFVI SW Configuration)**: a set of settings (Key:Value) that are applied/mapped to **NFVI** SW deployment.
```

Comment:
Suggest changing `NFVI Software Configuration` to `Cloud Infrastructure Software Configuration`